templates/addons/calico-dual-stack/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+    - https://docs.projectcalico.org/v3.22/manifests/calico-policy-only.yaml
```

Comment:
let's change this to a static file since the manifest isn't immutable, see #2149 

we're already getting calico-policy-only.yaml for ipv6 so you can just reuse the same one

Reply:
Do you recommend copying `calico-policy-only.yaml` to calico-dual-stack? I don't think we can reference it in kustomization.yaml from capi-ipv6 with a relative path from a different directory. AFAIK it only supports current or child directories.

Reply:
you can either copy `calico-policy-only.yaml` to calico-dual-stack or use ../calico-ipv6 as the base for dual stack and then patch on top of it (then you can also get rid of the azure-mtu patch)